infra/main.bicep

```diff
@@ -402,16 +400,16 @@ module searchRoleBackend 'core/security/role.bicep' = {
 
 module isolation 'network-isolation.bicep' = if (usePrivateEndpoint) {
   name: 'networks'
-  scope:  resourceGroup
+  scope: resourceGroup
   params: {
     location: location
     tags: tags
     resourceToken: resourceToken
-    vnetName: vnetName
+    vnetName: '${abbrs.virtualNetworks}${resourceToken}'
```

Comment:
Do we want to support using existing vnet?

Reply:
This PR is already really complicated, if they BYO VNET then they'd have to supply Private DNS zones, Private Endpoints, IP Subnets.. IMO we can look at that if it comes up

Reply:
I'd really appreciate the option of BYO VNET - My VNets, Subnets, Private DNS zones, and Expressroute resources are already in place.